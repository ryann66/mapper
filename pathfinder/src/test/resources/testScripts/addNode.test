CreateGraph g1
ListNodes g1
AddNode g1 n1
ListNodes g1
AddNode g1 n2
ListNodes g1