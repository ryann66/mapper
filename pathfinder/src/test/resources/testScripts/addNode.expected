created multigraph g1
g1 contains:
added node n1 to g1
g1 contains: n1
added node n2 to g1
g1 contains: n1 n2